src/app/shared/entity/external-machine-deployment.ts

```diff
@@ -127,4 +127,11 @@ export class ExternalMachineDeploymentSpecPatch {
 export class ExternalClusterMDPhase {
   state: ExternalClusterMDState;
   statusMessage?: string;
+  aks?: AKSState;
```

Comment:
[Move comment here](https://github.com/kubermatic/dashboard/pull/5038/files#diff-f7765865162f4d8038338babc7740768572ecf6bb8d0b7f436520beba8bd47efR133)  + update comment which shows that this option is available ONLY in AKS not for EKS/GKE